app/controllers/hosts_controller.rb

```diff
@@ -729,7 +729,7 @@ def clean_interfaces_attributes
     attributes = params[:host].dup
     if params[:host][:interfaces_attributes]
       attributes[:interfaces_attributes] = params[:host][:interfaces_attributes].dup.except(:created_at, :updated_at, :attrs)
-      attributes[:interfaces_attributes][:id] = nil
+      attributes[:interfaces_attributes][:id] = nil if attributes[:interfaces_attributes][:id]
```

Comment:
It'd be fine to add this above as `.dup.except(:id, :created_at, :updated_at, :attrs)`